gradle.properties

```diff
@@ -37,4 +37,5 @@ kotlin.code.style=official
 android.suppressUnsupportedCompileSdk=33
 
 # suppress Kotlin Multiplatform Projects are an "Alpha feature" warning
-kotlin.mpp.stability.nowarn=true
\ No newline at end of file
+kotlin.mpp.stability.nowarn=true
+kotlin.native.ignoreDisabledTargets=true
```

Comment:
Removes warnings about not being able to build iOS on linux, don't have strong opinions about it.

Reply:
I'm fine with this - but would prefer a comment stating this with a relevant link if there is one